examples/advanced/friendship-bot.py

```diff
@@ -33,7 +33,7 @@ def __init__(self):
         self.auto_reply_comment = "Automatic Reply: I cannot read your message because I'm busy now, will talk to you when I get back."
 
     async def on_friendship(self, friendship: Friendship):
-        administrator = bot.Contact.load('admin-id')
+        administrator = bot.Contact.load('self.contact_id')
```

Comment:
I guess you want to use `self.contact_id` instead of `'self.contact_id'`?

Reply:
It's a big shock to see a strange name.
So I just use self.contact_id.

Reply:
Oh yes, it's a string, my mistake.

My suggestion would use a better way for making this code to be great:

1. use a variable, like `ADMINISTRATOR_CONTACT_ID` to store this `administrator id`, defined at the beginning of the code file, or
2. use `filehelper` as this administrator by default, because the bot is always able to send message to `filehelper`.

Reply:
send message to filehelper instead print message on screen. Great!!!

Reply:
In my experience, I always create a room called `DevOps XXX` for the bot to sending log messages inside it.

Reply:
You're the boss. You're always right.
But It's just a novice project.